front/components/actions/github/GithubGetPullRequestActionDetails.tsx

```diff
@@ -0,0 +1,25 @@
+import { GithubIcon } from "@dust-tt/sparkle";
+import type { GithubGetPullRequestActionType } from "@dust-tt/types";
+
+import { ActionDetailsWrapper } from "@app/components/actions/ActionDetailsWrapper";
+import type { ActionDetailsComponentBaseProps } from "@app/components/actions/types";
+
+export function GithubGetPullRequestActionDetails({
```

Comment:
The "Github Actions" naming is somewhat confusing as it clashes with "Github Actions" the product... But I don't have a better suggestion

Reply:
Eventually it'll be called Platform Actions with a sub-menu "Github" anyway

Reply:
Yeah might be clearer 👍